src/app/sections/TelemetrySearchResults.svelte

```diff
@@ -115,21 +115,20 @@ li:hover {
     </div>
     {#if $searchResults.results.length}
         <ul>
-        {#each $searchResults.results as { name, probeType, description, id, searchID,
-        firstRelease, firstReleaseName }, i (id)}
+        {#each $searchResults.results as probe, i (probe.id)}
```

Comment:
code nit: could we destructure `probe` like `{#each $searchResults.results as {name, type, description, ...}, i (id) }`? I find it helps the readability in the block.

Reply:
There's 9 properties by my count which would make that a long line. And we're not using all properties in the block. We would also have to "restructure" all the properties back again in the `updateProbe` call (or only those we want in the store, but having what's in the API in the store is good, I think). I find having the object and the dot notation to access the properties very readable here.

Reply:
I think we're only using a handful specifically in the results, but I also recognize this is a matter of taste / eslint-in-react-land preference.

Reply:
I'm not typically a front-end dev but destructuring is preferred here so I'll update it. I also just learned that you can destructure just the properties you want.